client/components/ProjectView.tsx

```diff
@@ -40,25 +41,56 @@ const GET_QUERY_DATA = gql`
 
 export default function ProjectView({ selectedProject, projectLoading }: ProjectViewProps) {
     /** State requirments for this component */
-    const [queries, setQueries] = useState<ProjectQuery[]>();
+    const [queries, setQueries] = useState<ProjectQuery[]>([]);
+    const [days, setDays] = useState<1 | 7 | 30 | 365>(7);
```

Comment:
is it worth creating an enum for the available options?

Reply:
Can you also add some comments explaining what each piece of state represents? 

It's not immediately clear that `days` is the lookback period. Could even call this something like `lookbackWindowInDays` or `lookbackDays`

Reply:
I've added a type alias called `ChartSelectionDays` that is used to better type this variable across all components. I have also renamed some stuff. While doing that, I found all kinds of stuff that could be simplified and streamlined.